zebra-rpc/src/methods/get_block_template_rpcs/types/get_block_template.rs

```diff
@@ -9,6 +9,40 @@ use crate::methods::{
     GetBlockHash,
 };
 
+#[derive(Debug, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum GetBlockTemplateRequestMode {
+    Template,
+    Proposal,
+}
+
+impl Default for GetBlockTemplateRequestMode {
+    fn default() -> Self {
+        Self::Template
+    }
+}
+
+#[derive(Debug, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum GetBlockTemplateCapability {
+    LongPoll,
+    CoinbaseTxn,
+    CoinbaseValue,
+    Proposal,
+    ServerList,
+    WorkId,
```

Comment:
This list is missing the mutations:
https://en.bitcoin.it/wiki/BIP_0022#Block_Template_Request

This will fail if a client specifies a capability we don't know about.
Would it make sense to have an `UnknownCapability(String)` variant?

Reply:
I commented it out and used a `Vec<String>` instead in [`423f546`](https://github.com/ZcashFoundation/zebra/pull/5772/commits/423f546839628d32b6fd93e609d6a45506bb847a).